hbase-client/src/main/java/org/apache/hadoop/hbase/client/RpcRetryingCallerFactory.java

```diff
@@ -48,15 +44,15 @@ public RpcRetryingCallerFactory(Configuration conf, RetryingCallerInterceptor in
     startLogErrorsCnt = conf.getInt(AsyncProcess.START_LOG_ERRORS_AFTER_COUNT_KEY,
       AsyncProcess.DEFAULT_START_LOG_ERRORS_AFTER_COUNT);
     this.interceptor = interceptor;
-    enableBackPressure = conf.getBoolean(HConstants.ENABLE_CLIENT_BACKPRESSURE,
-      HConstants.DEFAULT_ENABLE_CLIENT_BACKPRESSURE);
   }
 
   /**
    * Set the tracker that should be used for tracking statistics about the server
+   * @deprecated Does nothing. Since 2.5.0, will be removed in 4.0.0.
```

Comment:
Remove this in 3.0.0 instead?

Reply:
This class is IA.Private so we are free to remove it at any time.